troposphere/static/js/components/modals/project/ProjectDeleteResourceModal.react.js

```diff
@@ -1,134 +0,0 @@
-/** @jsx React.DOM */
```

Comment:
Yep dead code.. We now have a 'DeleteConditionsModal' that says all resources must be moved/destroyed before deleting a project.